Assets/LeapMotion/Scripts/HandProxy.cs

```diff
@@ -44,11 +34,38 @@ public class HandProxy:
           handModels[i] = null;
         }
       }
+      parent.ActiveHandReps.Remove(this);
+    }
+
+    public override void AddModel(IHandModel model) {
+      if (handModels == null) {
+        handModels = new List<IHandModel>();
+      }
+      handModels.Add(model);
+      if (model.GetLeapHand() == null) {
+        model.SetLeapHand(MostRecentHand);
+        model.InitHand();
+        model.UpdateHand();
```

Comment:
There is a subtle bug here.  When Init is called, Update is called right after, even though Begin has not been called yet.  Begin should always be called before Update is called.